### PR TITLE
[OPAL-432] Fetch appropriate json schemas

### PR DESCRIPTION
**Pull request by iseessel**

### CODE REVIEW

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/products_controller.rb [hunk 1]
```diff
@@ -25,7 +25,11 @@ def index
 
   # GET /mnoe/jpi/v1/admin/products/id
   def show
-    @product = MnoEnterprise::Product.find_one(params[:id], DEPENDENCIES)
+    @product = MnoEnterprise::Product
```
Comment (BrunoChauvet) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/products_controller.rb]: Do we need to deal with the scenario where `editAction` is not specified?

Comment (iseessel) [reply api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/products_controller.rb]: @BrunoChauvet So there is guarding on the adapter bluesky for when the edit_action isn't specified -- nothing should break on mnoe when its not specified, but we may want to put a default here as well.

File: api/app/views/mno_enterprise/jpi/v1/admin/subscriptions/_subscription.json.jbuilder [hunk 2]
```diff
@@ -9,6 +9,7 @@ json.available_licenses subscription.available_licenses
 json.external_id subscription.external_id
 json.custom_data subscription.custom_data
 json.provisioning_data subscription.provisioning_data
+json.available_edit_actions subscription.available_edit_actions
```
Comment (BrunoChauvet) [inline api/app/views/mno_enterprise/jpi/v1/admin/subscriptions/_subscription.json.jbuilder]: Before we forget, this has been renamed to `available_actions`
https://github.com/maestrano/maestrano-hub/pull/972/files#diff-c8fbb65db37a91bbb5be88241f81e874R24

File: api/app/views/mno_enterprise/jpi/v1/subscriptions/_subscription.json.jbuilder [hunk 3]
```diff
@@ -9,6 +9,7 @@ json.available_licenses subscription.available_licenses
 json.external_id subscription.external_id
 json.custom_data subscription.custom_data
 json.provisioning_data subscription.provisioning_data
+json.available_edit_actions subscription.available_edit_actions
```
Comment (BrunoChauvet) [inline api/app/views/mno_enterprise/jpi/v1/subscriptions/_subscription.json.jbuilder]: Same comment as above

File: api/app/views/mno_enterprise/jpi/v1/admin/subscriptions/_subscription.json.jbuilder (hunk 2 above)
Comment (iseessel) [reply api/app/views/mno_enterprise/jpi/v1/admin/subscriptions/_subscription.json.jbuilder]: @BrunoChauvet Oops, yeah I had changed it but not pushed up the changes.

**Pull request merged**